### PR TITLE
PP-7409 Add test vs. live content for payment link created message

### PR DESCRIPTION
**Pull request by sfount**
Payment links created in test and live environments have very different
use cases; the wording for the payment link creation references the
payment link being "live" which conflicts with our use of this status
for accounts.

Add a separate path for test and live accounts to provide users
additional information if they are just testing.

**Payment link created in a test account**

Screenshot 2020-11-19 at 12 06 30

**Payment link created in a live account**

Screenshot 2020-11-19 at 12 08 33

### CODE REVIEW

**Comment by kaydale** [comment]
Looks good to me

File: app/views/payment-links/manage.njk [hunk 1]
```diff
@@ -12,10 +12,26 @@
...
 {% if flash.createPaymentLinkSuccess %}
-    {{ successNotification({
-      heading: "Your payment link is now live",
-      body: "Give this link to your users to collect payments for your service."
-    }) }}
+  {% if isTestGateway %}
+    {% set heading = "Your test payment link is ready to use" %}
+    {% set body %}
+      <span>
+        To collect payments from your users with this link you will need to
```
Comment (kbottla) [inline app/views/payment-links/manage.njk:19]:
```suggestion
        To collect payments from your users you will need to  ?
```

Comment (sfount) [reply app/views/payment-links/manage.njk:19]: As there's no line breaking content in this span everything in the block will be treated as the same line, they're on separate lines here for readability as the link in particular is long

Comment (kbottla) [reply app/views/payment-links/manage.njk:19]: I was trying to convey something using suggestion but clearly not done well :) . Do we need `with this link` as it won't be same link after live ?

Comment (sfount) [reply app/views/payment-links/manage.njk:19]: Ah! My bad, I thought the question mark was part of the suggestion 🤦

**Review by kbottla — APPROVED** [review]
LGTM!

File: app/views/payment-links/manage.njk (hunk 1 above)
Comment (iqbalgds) [reply app/views/payment-links/manage.njk]: Content has been updated.

**Review by kbottla — APPROVED** [review]
LGTM!

**Review by kbottla — APPROVED** [review]

**Pull request merged**